react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,37 @@
 import { isArray } from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (args.length === 0) {
+    return undefined;
+  }
+  if (args.length > 1) {
+    return Math.min(...args);
+  }
+  if (isArray(args[0])) {
+    return Math.min(...args[0]);
+  }
+
+  return Math.min(args);
 }
 
-export function copy() {
+export function copy(args) {
```

Comment:
**Mari de la Cámara:** 
> This copy function works fine! But I think you have to use spread because the idea of the exercise is that you learn the multiple uses of it.
So, you need to refactor copy function :( this time, using spread. You will see that it will be more readable and easy to understand!